lib/plugins/paragon-webpack-plugin/utils.js

```diff
@@ -0,0 +1,373 @@
+const { sources } = require('webpack');
```

Comment:
[proposal]: This file is quite large. I suggest dividing it into several files and creating a folder with utilities. I think it would be a good idea to follow this structure:

**./utils**

**htmlUtils.js**
- getDescendantByTag
- findScriptInsertionPoint
- findStylesheetInsertionPoint

**scriptUtils.js**
- minifyScript
- insertScriptContentsIntoDocument

**stylesheetUtils.js**
- insertStylesheetsIntoDocument

**assetUtils.js**
- findCoreCssAsset
- findThemeVariantCssAssets
- getCssAssetsFromCompilation

**scriptContentUtils.js**
- addToScriptContents
- generateScriptContents

**urlUtils.js**
- handleVersionSubstitution
- getParagonStylesheetUrls

Reply:
I agree and I did not the same order but let me know what you think :)

Reply:
Thank you! This looks great 💯